### ISSUE TITLE
Feature request: Spawn X amount of instances

### ISSUE THREAD

**Issue by unhiro** [jlplenio/crude-twitch-viewer-bot#91]
Hello,

First I'd like to start with thank you for making this tool it works great. The newest update works smoother than the last!

There are two features I would love to see added:

1. I would love to be able to set the max instances I would like to spawn and then just click a button and the program starts spawning them automatically. E.G.: I enter 45 into a text field delimiting the amount of viewers I want and then hit a spawn button that spawns one instance at a time and once that instance is viewing(or X amount of seconds between or timeout) then the next instance would spawn automatically.

2. This has already been mentioned but logging into accounts with user/pass that are saved in a file. (I think this would require each account be set to not have any 2 factor auth(email or phone) but that's something I would be willing to go through and deal with on my own before using them)

Otherwise it currently takes quite a bit of time to get it all running which isn't terrible but it would be a huge QoL.

**Comment by jlplenio** [comment]
1. I like this manual work to prevent automation and commercialization. So, I won't work on this.
Clicking a few times and spawning the viewers is not that inconvenient.
2. Logging in via token, chatting and following is finished in theory. But I have technical difficulties.
https://github.com/jlplenio/crude-twitch-viewer-bot/issues/40#issuecomment-1416810867

**Comment by jlplenio** [comment]
Good job on your coding progress! My first versions of CTVBot were based on selenium, I am trying to remember why I ultimately switched to playwright. I can give you my code to hide the navigator.webdriver and pass all tests at least on https://bot.sannysoft.com/. Do you use the chrome channel for playwright, Twitch dislikes chromium:
https://github.com/jlplenio/crude-twitch-viewer-bot/blob/c79b2fd6b2ac4e496345bad2633fe17cb064a5da/ctvbot/instance.py#L163

**Comment by unhiro** [comment]
I have tried that option in my Playwright version and my Selenium version passes all the tests on that website. I can't for the life of me find the other website I was testing on that shows your WebRTC IP and the time zone your machine is displaying and time zone your proxy is in(I'll update when I find it). I managed to fix the location stuff but not Playwright or Selenium leaking webrtc stuff. Twitch works perfectly fine until you hit the login button after entering a username and password. That's when it does a deeper check of the browser. I'd love to hear about any other options to hiding the browser
